families/supplychain/restapi/openapi.yaml

```diff
@@ -0,0 +1,398 @@
+swagger: '2.0'
+
+info:
+  version: "0.8.0"
+  title: Sawtooth Supplychain REST API
+  description: |
+    _This HTTP pragmatic REST API is built on top of Sawtooth Lake's existing
+    ZMQ/Protobuf infrastructure, simplifying SupplyChain client interactions
```

Comment:
We can implement it either way for this iteration. As long as we keep our data requests small additional data handling probably won't be an issue. The future will probably have this component become a state export client, so it can build and maintain it's own indexes for more efficient access and reporting.